server/client-config/routes.js

```diff
@@ -12,4 +34,23 @@ module.exports = app => {
     res.type('js')
     res.send(response)
   })
+
+  app.get('/ping', nocache, async (req, res) => {
+    try {
+      const exists = await DBExists()
+      if (!exists) {
+        sendResult(res, 404, 'database error')
+        return
+      }
+      const { Body } = await s3.getObject({ Key: 'healtcheck-file' }).promise()
+      if (!Body) {
+        sendResult(res, 404, 's3 connection error')
+        return
+      }
+
+      sendResult(res, 200, 'pong')
+    } catch (error) {
+      sendResult(res, 500, error)
```

Comment:
This would send the stack trace back as a response, which we should avoid as it can be a security risk.

We should return back `Internal Server Error` and log the error.